tests/integration/tier0/verify-latest-kernel-check-with-failed-repoquery/test_verify_kernel_check_passes.py

```diff
@@ -0,0 +1,27 @@
+def test_verify_latest_kernel_check_passes_with_failed_repoquery(shell, convert2rhel):
+    """
+    This test verifies, that failed repoquery is handled correctly.
+    Failed repoquery subsequently caused the latest kernel check to fail.
+    Introduced fixes should get the process past the latest kernel check.
+    """
+
+    repopath = "/etc/yum.repos.d/beaker.repo"
+
+    # Add a tainted repository with intentionally bad baseurl so the repoquery fails successfully.
+    assert shell(f"touch {repopath}").returncode == 0
+    assert (
+        shell(
+            "printf '[beaker-client]\nname=Beaker Client - Fedora36\nbaseurl=http://download.eng.bos.redhat.com/\nenabled=1\ngpgcheck=0' > /etc/yum.repos.d/beaker.repo"
+        )
+    ).returncode == 0
```

Comment:
A simple question here... Do we really need this to be a shell call only to assert that it created successfully the file? 

I'm thinking more towards using the `with open(...)` in python to make it more readable, but I'm not sure if using a shell here to assert the file creation is more correct than a python statement.

Reply:
I'd like to assert, the repofile was created successfully.
But you are right, I may do it more in a python way, and assert a simple `cat` to check for the repofile.